src/features/chart/components/Chart.js

```diff
@@ -26,23 +26,65 @@ const tenors = [
 ];
 
 export const Chart = () => {
-  const [active, setActive] = useState("5y");
-  // const chartData = stock.map(data => ({close:data.close, date:moment(data.close).format('lll') }))
+  const [chartRange, setChartRange] = useState("5y");
   const dispatch = useDispatch();
   const { selectedChartData } = useSelector(state => state.chartData);
   const { selectedStockTicker } = useSelector(state => state.stockTickerData);
   const updateChartRange = stock => dispatch(updateChartAction(stock));
   const onClickHandler = event => {
     updateChartRange(event.target.value);
-    setActive(event.target.value);
+    setChartRange(event.target.value);
+  };
+
+  const formatChartData = () => {
+    const formatDate = isoDate => {
+      const date = new Date(isoDate);
+      switch (chartRange) {
+        case "max":
+          return Intl.DateTimeFormat("en-US", {
+            year: "2-digit",
+            month: "short"
+          }).format(date);
+        case "5y":
+          return Intl.DateTimeFormat("en-US", {
+            year: "2-digit",
+            month: "short"
+          }).format(date);
+        case "1y":
+          return Intl.DateTimeFormat("en-US", { month: "short" }).format(date);
+        case "1m":
+          return Intl.DateTimeFormat("en-US", {
+            year: "2-digit",
+            month: "2-digit"
```

Comment:
changing '1m' month format to 'short' might look nicer

Reply:
Agreed - have changed